cg/apps/demultiplex/sample_sheet/create.py

```diff
@@ -12,11 +12,11 @@
 
 
 def create_sample_sheet(
-    bcl_converter: str,
+    bcl_converter: Literal[BclConverter.BCL2FASTQ, BclConverter.DRAGEN],
```

Comment:
Dragen is the hardware calling the BCLconvert software. In a future PR we should address that and change the constant.

```suggestion
    bcl_converter: Literal[BclConverter.BCL2FASTQ, BclConverter.BCLCONVERT],
```

Reply:
or should we do this:
```
class BclConverter(StrEnum):
    """Define the BCL converter."""

    DRAGEN: str = "bclconvert"
```